SPECS/pytorch/pytorch.spec

```diff
@@ -0,0 +1,92 @@
+%global _empty_manifest_terminate_build 0
+Summary:        Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+Name:           pytorch
+Version:        1.13.1
+Release:        1%{?dist}
+License:        BSD-3-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://pytorch.org/
+Source0:        https://github.com/pytorch/pytorch/releases/download/v%{version}/%{name}-v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+# Use the generate_source_tarball.sh script to create a tarball of submodules during version updates.
+Source1:        %{name}-%{version}-submodules.tar.gz
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  python3-PyYAML
+BuildRequires:  python3-devel
+BuildRequires:  python3-filelock
+BuildRequires:  python3-hypothesis
+BuildRequires:  python3-jinja2
+BuildRequires:  python3-numpy
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-typing-extensions
+
+%description
+PyTorch is a Python package that provides two high-level features:
+- Tensor computation (like NumPy) with strong GPU acceleration
+- Deep neural networks built on a tape-based autograd system
+You can reuse your favorite Python packages such as NumPy, SciPy and Cython to extend PyTorch when needed.
+
+%package -n     python3-pytorch
+Summary:        Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+Requires:       python3-filelock
+Requires:       python3-numpy
+Requires:       python3-typing-extensions
+
+%description -n python3-pytorch
+PyTorch is a Python package that provides two high-level features:
+- Tensor computation (like NumPy) with strong GPU acceleration
+- Deep neural networks built on a tape-based autograd system
+You can reuse your favorite Python packages such as NumPy, SciPy and Cython to extend PyTorch when needed.
+
+%package -n python3-pytorch-doc
+Summary:        Development documents and examples for torch
+
+%description -n python3-pytorch-doc
+PyTorch is a Python package that provides two high-level features:
+- Tensor computation (like NumPy) with strong GPU acceleration
+- Deep neural networks built on a tape-based autograd system
+You can reuse your favorite Python packages such as NumPy, SciPy and Cython to extend PyTorch when needed.
+
+%prep
+%autosetup -a 1
+
+%build
+export USE_CUDA=0
+export BUILD_CAFFE2=0
+%py3_build
+
+%install
+%py3_install
+install -d -m755 %{buildroot}/%{_pkgdocdir}
+
+cp -arf docs %{buildroot}/%{_pkgdocdir}
+
+%files -n python3-pytorch
```

Comment:
License needs to be packaged here, and in the doc subpackage as well

Reply:
Done. Ran a local build after this change to confirm the package built successfully.